src/core/core.adapters.js

```diff
@@ -8,7 +8,7 @@
  * @return {*}
  */
 function abstract() {
-  throw new Error('This method is not implemented: either no adapter can be found or an incomplete integration was provided.');
+  throw new Error('This method is not implemented: Check that a complete date adapter is provided.');
```

Comment:
Wouldnt it be better to use a variable for the type of adapter so that if in the future extra adapters are needed/added you dont have to change it again and can just pass that adapter name

Reply:
IMO that can be left for the future, trivial to update with a variable when its actually needed